zenodo/modules/records/templates/zenodo_records/record_detail.html

```diff
@@ -73,7 +73,7 @@ <h1 itemprop="name">{{ record.title }}</h1>
         <h5>{{_('Thesis supervisor(s)')}}</h5>
         <p>{{ authors(record.thesis.supervisors) }}</p>
       {%- endif %}
-      <p itemprop="description">{{ record.description|safe }}</p>
+      <p>{{ record.description|safe }}</p>
```

Comment:
@lniensen is it OK to pipe the JSON (string) from a serializer output through ``safe`` here?